client/src/components/TextArea/TextArea.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./TextArea.css";
+
+const TextArea = props => (
+  <div>
+    <label className={props.titleClassList}>{props.title}</label>
+    <textarea className={props.textAreaClassList} onChange={props.onChange}>
+      {props.innerText}
```

Comment:
Can you think of a better name than `innerText`? `value` is a good one that is more inline with convention